playground/backend/internal/logger/std_handler.go

```diff
@@ -0,0 +1,77 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"fmt"
+	"log"
+)
+
+type StdHandler struct {
```

Comment:
Is this handler used only to log messages using standard `log`? If yes I suggest adding some comments (or maybe renaming) about that so this point will be more understandable.

Reply:
@KhaninArtur @ilya-kozyrev what do you think?

Reply:
I agree, it is better to add some comments for it

Reply:
I see that you've added comments on the place where you are setting up the logger handler (`SetupLogger()` method). Now it is more clear. However, this struct is still not so clear for me (moreover if I don't know about the `SetupLogger()` method). So could you please add a comment here too?

Reply:
Done.